FEM/rf_msp_new.cpp

```diff
@@ -1239,6 +1295,7 @@ CSolidProperties::~CSolidProperties()
     data_Density = NULL;
     data_Youngs = NULL;
     data_Plasticity = NULL;
+    data_Plasticity_joint = NULL;
```

Comment:
Please copy this line to the constructor as well, i.e before line 1239. The uninitialized pointer causes problem in the destructor under MPI and PETSc.